pkg/generic/thriftidl_provider.go

```diff
@@ -17,19 +17,28 @@
 package generic
 
 import (
+	"context"
 	"fmt"
+	"os"
 	"path/filepath"
 	"sync"
 
+	dthrift "github.com/cloudwego/dynamicgo/thrift"
 	"github.com/cloudwego/thriftgo/parser"
 
 	"github.com/cloudwego/kitex/pkg/generic/descriptor"
 	"github.com/cloudwego/kitex/pkg/generic/thrift"
 )
 
+const (
+	GoTagDisabled    = "KITEX_GENERIC_GOTAG_ALIAS_DISABLED"
+	UseDynamicgoConv = "KITEX_GENERIC_USE_DYNAMICGO_CONVERSION"
```

Comment:
You mean user need to specify the env to use dynamicgo? Not by default？

Reply:
@jayantxie This is because we can't know if user wants to use which generic call when parsing idl (like `NewThriftFileProvider`). I don't want to make dynamicgo descriptor when it's not needed (like when user wants to user binary generic call).

Reply:
fixed